test/src/test/java/org/corfudb/runtime/view/StreamViewTest.java

```diff
@@ -385,4 +392,67 @@ public void prefixTrimThrowsException()
         assertThatThrownBy(() -> sv.next())
                 .isInstanceOf(TrimmedException.class);
     }
+
+    @Test
+    public void testPrevious1() {
```

Comment:
Change the test name to be more descriptive and add comments describing the test.

Reply:
Changed the test name, i think a description would be superfluous with the new test name.